spec/test-data/translations/strings-with-escape-character-in-root-context.strings

```diff
@@ -0,0 +1,6 @@
+/* This file has characters root context (not in a comment, or key, or value), which is not a valid `.strings` syntax */
+"key" = "value";
+"other key" = "other value";
+\\
```

Comment:
@AliSoftware is this what you were thinking of when talking about escaped characters in the root context?

Is there any other example we should add a it's own `.strings` file to ensure the helper catches it?

Thanks

Reply:
Yes that's what I was thinking about when I talked about escaped characters in the root context.
(And if I'm not mistaken, such a `\\` in such a position in the `.strings` file is not supposed to be valid)

Reply:
> And if I'm not mistaken, such a `\\` in such a position in the `.strings` file is not supposed to be valid

That seems to be the case:

![image](https://user-images.githubusercontent.com/1218433/162664253-99a135a5-85cd-45fd-9d0b-eca100b04d4e.png)

However, if I move the `\\` down in the file, I don't get the error 🤔 

<img width="1974" alt="image" src="https://user-images.githubusercontent.com/1218433/162664302-9d8eb5cd-1981-4862-a5a5-927afd89e857.png">

Not sure whether that's an Xcode issue or not 🤔

Reply:
I've seen `plutil` consisting an invalid Plist (or .strings) file to be valid but truncated before (and I'm assuming Xcode might rely on the same logic as plutil for this?). Especially I remember a while ago having a strings file missing a semicolon at the end of one line and `plutil -lint` telling me that the file was linting OK… But `plutil -p` just stopping parsing after that line and totally ignoring any content after that missing semicolon like if no content existed after the syntax error.

I wonder what you get, and especially if there might be missing keys in the output, when running `plutil -p` on that second example that Xcode fail to warn you about … could it be the same bug? If so, our parser might be smarter at detecting syntax errors than Xcode's / `plutil` 😅